#
# Lane Model
#
Header header
float32[] poly
float32 x_min
float32 x_max